content/docs/examples/httpendpoints.md

```diff
@@ -11,36 +11,116 @@
 # Build an HTTP API Endpoint
 {{<header_divider>}}
 
-HTTP endpoints have become one of the most ubiquitous ways of delivering software to end users. Cdev provides a simple interface for creating HTTP APIs and connecting them to `Serverless Functions`. 
+`HTTP` endpoints have become one of the most ubiquitous ways of delivering software to end users. Cdev provides a simple interface for creating `HTTP APIs` and connecting them to `Serverless Functions`. You can find more information about `HTTP` in the **[official documentation](https://httpwg.org/)**.
+
+{{<tool_tip key="info" summary="Set Up">}}
+It is assumed that you have already installed the Cdev SDK, which has a pre-filled src/hello_world/resources.py file. This example starts with an empty src/hello_world/resources.py file. You can empty the contents of your file or replace the neccessary lines of code as you go along.
+{{</tool_tip>}}
 
 {{<break 1>}}
 ## Create an API
-{{<codesnippet `/source_code/http_examples/simple_api_example.py`>}}
+
+Open the `src/hello_world/resources.py` file and start by importing the following resources:
+{{<codesnippet `/source_code/http_examples/simple_api_example_resources.py`>}}
+
+
+{{<break 1>}}
+## Create a route
+Next, add lines `lines 8, 10 and 12` to define your project, `API` and create your route.
+{{<codesnippet `/source_code/http_examples/simple_api_example_create_route.py`>}}
 
 
 {{<break 1>}}
-## Create routes
-{{<codesnippet `/source_code/http_examples/simple_api_example_get.py`>}}
+## Integrate GET route with a Serverless Function
+Now, intergrate your route with a [Serverless Function](/docs/examples/functions/) by adding `lines 10 - 35`.
+{{<codesnippet `/source_code/http_examples/simple_api_example_get_serverless.py`>}}
 
 
 {{<break 1>}}
-## Integrate GET route with a Serverless function
-{{<codesnippet `/source_code/http_examples/simple_api_example_integration.py`>}}
+### Register your GET route
+Open the `src/cdev_project.py` file and register your `GET` route by replacing `lines 15-17` with the following:
```

Comment:
I think we can remove this part. This part should be setup from the `quick-start` template.

Reply:
Thanks Daniel.  I have removed that section from the Get and Post route examples.